web/cobrands/sass/_base.scss

```diff
@@ -1087,6 +1092,34 @@ footer {
   }
 }
 
+.footer-main-nav {
+  padding: $footer-main-nav-padding;
+  background-color: $footer-main-nav-background;
+  .nav-footer {
+    display: flex;
+    column-gap: 2rem;
+    row-gap: 1rem;
+    font-size: $footer-main-nav-font-size;
+    justify-content: center;
+    margin: 0;
+    flex-wrap: wrap;
+  
+    li {
+        list-style: none;
+        margin-bottom: 0;
+    }
+
```

Comment:
I think this bit needs adding back in:

```suggestion
    .frontpage & {
            // In frontpage users can't interact with 'Report a problem' link
            li:first-child {
                display: none;
            }
        }
```


<img width="463" alt="image" src="https://github.com/user-attachments/assets/9240f763-dfa2-45d4-91c9-e32597d4eaa4">

Reply:
This bit shouldn't be needed anymore because of this: https://github.com/mysociety/fixmystreet/pull/5122